qrenderdoc/Windows/Dialogs/CaptureDialog.ui

```diff
@@ -410,48 +455,69 @@ Click here to set up Vulkan capture.</string>
       </item>
       <item>
        <widget class="QCheckBox" name="CaptureCallstacks">
+        <property name="toolTip">
+         <string>Collect a callstack on every API call</string>
+        </property>
         <property name="text">
          <string>Collect Callstacks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CaptureCallstacksOnlyDraws">
+        <property name="toolTip">
+         <string>Only collect callstack on every API call</string>
```

Comment:
This isn't quite right, this option is about only collecting callstacks for 'drawcall' type calls. Things like draws, copies, resolves, etc. The C# tooltip was;

> Only collect callstacks on 'drawcall' level api calls